deeppavlov/core/data/utils.py

```diff
@@ -55,29 +55,40 @@ def simple_download(url: str, destination: [Path, str]):
 
     destination = Path(destination)
     destination.parent.mkdir(parents=True, exist_ok=True)
+    temporary = destination.with_suffix(destination.suffix + '.part')
 
     headers = {'dp-token': get_download_token()}
     r = requests.get(url, stream=True, headers=headers)
     total_length = int(r.headers.get('content-length', 0))
 
     log.info('Downloading from {} to {}'.format(url, destination))
-    with destination.open('wb') as f, tqdm(total=total_length, unit='B', unit_scale=True) as pbar:
+    with temporary.open('ab') as f:
         done = False
-        downloaded = 0
-        while not done:
-            for chunk in r.iter_content(chunk_size=CHUNK):
-                if chunk:  # filter out keep-alive new chunks
-                    downloaded += len(chunk)
-                    pbar.update(len(chunk))
-                    f.write(chunk)
-            if downloaded < total_length:
-                log.warn(f'Download stopped abruptly, trying to resume from {downloaded} to reach {total_length}')
-                headers['Range'] = f'bytes={downloaded}-'
-                r = requests.get(url, headers=headers, stream=True)
-                if total_length - downloaded != int(r.headers['content-length']):
-                    raise RuntimeError('It looks like the server does not support resuming downloads')
-            else:
-                done = True
+        downloaded = f.tell()
+        if downloaded != 0:
+            log.warn(f'Found a partial download {temporary}')
+        with tqdm(initial=downloaded, total=total_length, unit='B', unit_scale=True) as pbar:
+            while not done:
+                if downloaded != 0:
+                    log.warn(f'Download stopped abruptly, trying to resume from {downloaded} '
+                             f'to reach {total_length}')
+                    headers['Range'] = f'bytes={downloaded}-'
+                    r = requests.get(url, headers=headers, stream=True)
+                    if 'content-length' not in r.headers or \
+                            total_length - downloaded != int(r.headers['content-length']):
+                        raise RuntimeError(f'It looks like the server does not support resuming '
+                                           f'downloads. Please remove {temporary} and try again')
```

Comment:
I've sent a [PR](https://github.com/Huawei-MRC-OSI/DeepPavlov/pull/1) about not asking to remove a temporary file